frontend/android/app/src/main/java/elte/moneyshare/view/AddSpendingFragment.kt

```diff
@@ -128,11 +130,29 @@ class AddSpendingFragment : Fragment() {
                     selectMembersRecyclerView.adapter = adapter
                 }
 
+                selectEveryoneButton.invisible()
+                selectNooneButton.invisible()
+
                 nextButton.invisible()
                 addButton.visible()
             }
         }
 
+        selectEveryoneButton.setOnClickListener {
+            val selectedIds: ArrayList<Int> = ArrayList()
+            viewModel.currentGroupData?.members?.forEach {
+                selectedIds.add( it.id )
+            }
+            (selectMembersRecyclerView.adapter as SelectMembersRecyclerViewAdapter).selectedIds = selectedIds
+            adapter?.notifyDataSetChanged()
```

Comment:
itt ugyanúgy ahogy elkéred az adaptert a view-tól előtte levő sorban megteheted itt is és nem kell kitenni változóba.
(selectMembersRecyclerView.adapter as SelectMembersRecyclerViewAdapter).let{
ide írd a 2 műveletet az adapteren
}